src/views/settings/components/utils.js

```diff
@@ -14,8 +14,13 @@ export const differenceObject = (object, base) => {
 	function changes(object, base) {
 		return transform(object, (result, value, key) => {
 			if (!isEqual(value, base[key])) {
-				// eslint-disable-next-line no-param-reassign
-				result[key] = isObject(value) && isObject(base[key]) ? changes(value, base[key]) : value;
+				if (key === 'zimbraPrefMailTrustedSenderList') {
```

Comment:
you could bring this condition to the previous if to avoid nesting

Reply:
we need to check isEqual(value, base[key]) for both cases so bring this condition to the previous will not work.